app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command_name: str) -> None:
+    command_name = command_name.lower()
+    command_tokens = command_name.split(" ")
+
+    if "" in command_tokens:
+        for _ in range(command_tokens.count("")):
+            command_tokens.remove("")
+
+    if len(command_tokens) != 3:
+        raise SyntaxError("File-copy command must consist of three tokens")
+    if command_tokens[0] != "cp":
+        raise SyntaxError("File-copy command is not recognized")
+    if command_tokens[1] == command_tokens[2]:
+        return
+
+    with open(command_tokens[1], "r") as file_in, \
```

Comment:
use () instead of \

Reply:
Sorry, but the parentheses give me an error -- please, see the attached picture
![скрін_6](https://github.com/mate-academy/py-copy-file/assets/75977390/411a5623-88f3-4b04-91d0-e73f5f7c8e64)

Reply:
-- PyCharm says: "Unexpected expression syntax".

Could it be because of my obsolete Python version? -- Have 3.8

Reply:
Problem in your version of Python. You can use parentheses only from python 3.10

Reply:
I would have gladly upgraded but my PyCharm's version is 2020.1.5 and it only supports the Python versions up to, and including, 3.9.

So, kindly, am unable to avoid the awkward syntax in this case